string status_flag
float64 distance
